Dfe.Data.SearchPrototype.Data/appsettings.json

```diff
@@ -4,6 +4,6 @@
     "indexName": "establishments",
     "apiKey": "populate me via secrets:)"
   },
-  "filePath": "C:\\SearchPrototypeData\\GIASextract\\edubasealldata20240807.csv"
-
+  "filePath": "C:\\SearchPrototypeData\\GIASextract\\edubasealldata20241031.csv",
+  "PostCodeAPIEndpoint": "https://api.postcodes.io/"
```

Comment:
`minor` Should these PostCodeAPI be a `PostCodeApiOptions` so that it can be extended if keys / url parts need to be passed it's easier?

Reply:
Could do - but this is just a noddy dev way of getting some data into our AI search index and not meant to be used in production